src/components/calcite-action-menu/calcite-action-menu.scss

```diff
@@ -1,23 +1,30 @@
 :host {
-  @extend %component-host;
```

Comment:
Is the host/spacing no longer needed here? i know it has some box sizing stuff.

Reply:
I added the styles from %component-host via tailwind, so that should all be good.